.github/ISSUE_TEMPLATE/Epic.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Epic template
+about: Template for creating epics
+title: "[EPIC]"
+labels: Epic
+assignees: ''
+
+---
+
+<!--
+An Epic should define a higher-level package of work or project iteration that, once finished, makes sense to merge in core.
+-->
+## Description ##
+<!-- The what -->
+
+## Rationale ##
+<!-- The why -->
+
+## Stakeholders ##
+<!-- The who. This could include developers, designers, accessibility folks, testers, devrel, PMs... -->
+
+## Dependencies & Other Context ##
+<!--
+Link here any relevant Overview issues or posts that provide context
+-->
+
+## Acceptance Criteria & Required Scope ##
+<!--
+Please include here all the necessary bits and pieces to launch this package of work in a major WordPress release.
+It can include acceptance criteria in text form, but also the list of stories/issues that comprise the Epic.
+-->
+- [ ] 
+- [ ] 
+- [ ] 
+
+## Nice-to-haves ##
```

Comment:
I don't think we need this section. Maybe "Follow-ups" to not loose the additional stuff not scoped originally?